resources/views/cart/queries/cart.graphql

```diff
@@ -3,6 +3,7 @@ total_quantity
 items {
     id
     quantity
+    qty_backordered
```

Comment:
Is this the [value added in Compadre](https://github.com/rapidez/magento2-compadre/pull/3)? 
If so it must be wrapped in the Compadre version check with the [module.xml version](https://github.com/rapidez/magento2-compadre/blob/master/src/etc/module.xml#L4) to prevent errors when it is not installed/updated.

In your code you'll need to account for the key being missing